models/rarible/polygon/rarible_polygon_events.sql

```diff
@@ -0,0 +1,178 @@
+    post_hook='{{ expose_spells(\'["polygon"]\'
+                              "project",
+                              "rarible",
+                              \'["springzh"]\') }}'
+    )
+}}
+
+{% set nft_start_date = "2022-02-23" %}
+
+WITH trades AS (
+    select 'buy' AS trade_category,
+        evt_block_time,
+        evt_block_number,
+        evt_tx_hash,
+        contract_address,
+        evt_index,
+        'Trade' AS evt_type,
+        rightMaker AS buyer,
+        leftMaker AS seller,
+        '0x' || right(substring(leftAsset:data, 3, 64), 40) AS nft_contract_address,
+        CAST(bytea2numeric_v3(substr(leftAsset:data, 3 + 64, 64)) AS string) AS token_id,
+        newRightFill AS number_of_items,
+        CASE WHEN leftAsset:assetClass = '0x73ad2146' THEN 'erc721' ELSE 'erc1155' END AS token_standard, -- 0x73ad2146: erc721; 0x973bb640: erc1155
+        CASE WHEN rightAsset:assetClass = '0xaaaebeba' THEN '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270'
+            ELSE '0x' || right(substring(rightAsset:data, 3, 64), 40)
+        END AS currency_contract,
+        newLeftFill AS amount_raw
+    FROM {{ source ('rarible_polygon', 'Exchange_evt_Match') }}
+    WHERE rightAsset:assetClass in ('0xaaaebeba', '0x8ae85d84') -- 0xaaaebeba: MATIC; 0x8ae85d84: ERC20 TOKEN
+        {% if not is_incremental() %}
+        AND evt_block_time >= '{{nft_start_date}}'
+        {% endif %}
+        {% if is_incremental() %}
+        AND evt_block_time >= date_trunc("day", now() - interval '1 week')
+        {% endif %}
+
+    UNION ALL
+
+    select 'sell' AS trade_category,
+        evt_block_time,
+        evt_block_number,
+        evt_tx_hash,
+        contract_address,
+        evt_index,
+        'Trade' AS evt_type,
+        leftMaker AS buyer,
+        rightMaker AS seller,
+        '0x' || right(substring(rightAsset:data, 3, 64), 40) AS nft_contract_address,
+        CAST(bytea2numeric_v3(substr(rightAsset:data, 3 + 64, 64)) AS string) AS token_id,
+        newLeftFill AS number_of_items,
+        CASE WHEN rightAsset:assetClass = '0x73ad2146' THEN 'erc721' ELSE 'erc1155' END AS token_standard, -- 0x73ad2146: erc721; 0x973bb640: erc1155
+        CASE WHEN leftAsset:assetClass = '0xaaaebeba' THEN '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270'
+            ELSE '0x' || right(substring(leftAsset:data, 3, 64), 40)
+        END AS currency_contract,
+        newRightFill AS amount_raw
+    FROM {{ source ('rarible_polygon', 'Exchange_evt_Match') }}
+    WHERE leftAsset:assetClass in ('0xaaaebeba', '0x8ae85d84') -- 0xaaaebeba: MATIC; 0x8ae85d84: ERC20 TOKEN
+        {% if not is_incremental() %}
+        AND evt_block_time >= '{{nft_start_date}}'
+        {% endif %}
+        {% if is_incremental() %}
+        AND evt_block_time >= date_trunc("day", now() - interval '1 week')
+        {% endif %}
+),
+
+contract_list as (
+    SELECT distinct nft_contract_address
+    FROM trades
+),
+
+mints AS (
+    SELECT 'mint' AS trade_category,
```

Comment:
I think it's better to remove the mints setup (as in other PRs).
Getting all the distinct contracts traded on rarible is not correct and will result in duplicates with other models. 
Since we're not able to add any differentiating information (mint price, artist,...) we can just remove this.